public/app/plugins/panel/flamegraph/flamegraphV2/components/TopTable/FlameGraphTopTableContainer.tsx

```diff
@@ -72,6 +96,53 @@ function buildTableDataFrame(
     table[label].total = table[label].total ? table[label].total + value : value;
   }
 
+  const actionFieldTableConfig: TableFieldOptions = {
+    filterable: false,
+    width: 61,
+    hideHeader: true,
+    cellOptions: {
+      type: TableCellDisplayMode.Custom,
+      // @ts-ignore this isn't typed property from Cue
+      cellComponent: (props: CustomCellRendererProps) => {
```

Comment:
This will need to rebuild the dataframe each time any of the search or sandwich items change which is not ideal.